.github/CODEOWNERS

```diff
@@ -8,6 +8,9 @@
 # Catch all
 /sdk/                                               @joshfree @jonathangiles
 
+# BOM
+/sdk/bom/                                           @alzimmermsft @jonathangiles
```

Comment:
You need to update this file with the new path

Reply:
Fantastic catch!

Reply:
Updated